README.md

```diff
@@ -25,9 +25,8 @@ Setup Backend:
 3. Create local virtual environment: `python3 -m venv .venv` (or use `python`)
 4. Activate virtual environment: `source .venv/bin/activate`
 5. Install dependencies: `pip install -r requirements.txt`
-6. Install NLTK dependencies: `python3 -m nltk.downloader punkt` and `python3 -m nltk.downloader stopwords` (if you are running into an error, try running `pip install certifi /Applications/Python\ {PYTHON_VERSION}/Install\ Certificates.command` first and then re-installing the NLTK dependencies. Make sure {PYTHON_VERSION} is the same as what exists in your File Explorer!)
```

Comment:
Can we keep this step in the README? It's helpful since I'm assuming a developer still needs to manually run `python3 -m nltk.downloader vader_lexicon` when developing locally.

Reply:
Oh yeah I didn't realize we still needed it.